cumulusci/tasks/metadata/package.py

```diff
@@ -215,11 +215,38 @@ def render_xml(self):
         output.append("    </types>")
         return output
 
+    def strip_folder(self,component_list):
```

Comment:
Could you install the precommit checks (`pre-commit install`) in the repo so you'll get `black` formatting for Python code?